### PR TITLE
Feature/core 10943 geo tree fix default key for subcategories

### PR DESCRIPTION
**Pull request by irmakyz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 36.0.0
+Changed:
+- `GeoTree`: `keyForSubcategory` is changed as 'subcategories'.
```
Comment (Navas23) [inline CHANGELOG.md:5]: Let's say we only changed the default

Comment (irmakyz) [reply CHANGELOG.md:5]: - [ ] update changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 36.0.0
+Changed:
+- `GeoTree`, `GeoTreeItem`: Default value of `keyForSubcategory` is changed.
```
Comment (Navas23) [inline CHANGELOG.md:5]: Specify the change, so people reading the changelog does not have to check the code

Comment (irmakyz) [reply CHANGELOG.md:5]: - [ ] specify the change

**Review by Navas23 — APPROVED** [review]

**Pull request merged**